js/main.js

```diff
@@ -58,3 +65,46 @@ function searchChamps(event) {
     }
   }
 }
+
+function addChamps(event) {
+  var target = event.target.parentNode.parentNode.getAttribute('champ-name');
+  for (let i = 0; i < allChampions.length; i++) {
+    if (target === allChampions[i].name.toLowerCase()) {
+      var img = document.createElement('img');
+      var div = document.createElement('div');
+      var overlay = document.createElement('div');
+      var remove = document.createElement('img');
+      img.setAttribute('src', allChampions[i].big_image_url);
+      div.className = 'comp-img';
+      div.setAttribute('champ-name', allChampions[i].name.toLowerCase());
+      div.appendChild(img);
+      overlay.className = 'overlay-remove';
+      remove.setAttribute('src', 'images/remove.png');
+      overlay.appendChild(remove);
+      div.appendChild(overlay);
+      $comparisonList.appendChild(div);
```

Comment:
Could you possibly make this into a function that generates a DOM tree and returns it, so you can append the DOM tree where the function is called? In general, when you have a function that generates a DOM tree, you want to `return` the DOM tree instead of appending it directly to the page, so that the function is reusable if you need a DOM tree like that in another place in the application in the future.